crates/bevy_app/src/plugin_group.rs

```diff
@@ -27,6 +27,11 @@ impl PluginGroup for PluginGroupBuilder {
     }
 }
 
+// Helper method to get the `TypeId` of a value without having to name its type.
```

Comment:
```suggestion
/// Helper method to get the [`TypeId`] of a value without having to name its type.
```

Converting the comment into a doc comment will make it show up in the Rust Analyzer preview. (Even if not in `rustdoc`.)